Resources/Prototypes/XenoArch/Effects/normal_effects.yml

```diff
@@ -89,7 +89,7 @@
     - id: FoodPacketPopcornTrash
       prob: 0.1
       orGroup: Trash
-    - id: FoodPacketRaisinsTrash
+    - id: PlushieScientist
```

Comment:
Why are you replacing this?

Reply:
ah. I want replace random toy in that spawn rate. But I made a mistake with the line